package.json

```diff
@@ -121,7 +121,7 @@
             ]
         },
         "transformIgnorePatterns": [
-            "node_modules/(?!(@ckeditor|lodash-es)/)"
+            "node_modules/(?!(@ckeditor|ckeditor5|lodash-es)/)"
```

Comment:
See https://github.com/ckeditor/ckeditor5/issues/9554 there seems to be a strange new core packages just called `ckeditor5` which also need to transpiled